src/Type/BaseDoctrineORMSerializationType.php

```diff
@@ -145,8 +145,8 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
 
                 if (isset($associationMetadata['joinColumns']['nullable'])) {
                     $nullable = $associationMetadata['joinColumns']['nullable'];
-                } elseif (isset($associationMetadata['inverseJoinColumns']['nullable'])) {
-                    $nullable = $associationMetadata['inverseJoinColumns']['nullable'];
+                } elseif (isset($associationMetadata['joinTable']['inverseJoinColumns']['nullable'])) {
```

Comment:
Looking at the doctrine/orm code `inverseJoinColumns` only exists in the joinTable key.
This was now reported by phpstan since I added typed to the associationMapping in doctrine.